core/src/main/scala/org/broadinstitute/dsde/rawls/util/OpenCensusDBIOUtils.scala

```diff
@@ -19,4 +19,16 @@ object OpenCensusDBIOUtils {
       endSpan(span, Status.OK)
       DBIO.successful(0)
     }
+
+  def traceReadOnlyDBIOWithParent[T](name: String, parentSpan: Span, failureStatus: Throwable => Status = (_: Throwable) => Status.UNKNOWN)(f: Span => ReadAction[T])(implicit executor: ExecutionContext): ReadAction[T] =
+    traceReadOnlyDBIOSpan(startSpanWithParent(name, parentSpan), failureStatus)(f)
+
+  def traceReadOnlyDBIO[T](name: String, failureStatus: Throwable => Status = (_: Throwable) => Status.UNKNOWN)(f: Span => ReadAction[T])(implicit executor: ExecutionContext): ReadAction[T] =
+    traceReadOnlyDBIOSpan(startSpan(name), failureStatus)(f)
+
+  private def traceReadOnlyDBIOSpan[T](span: Span, failureStatus: Throwable => Status)(f: Span => ReadAction[T])(implicit executor: ExecutionContext): ReadAction[T] =
+    f(span).cleanUp { _ =>
+      endSpan(span, Status.OK)
+      DBIO.successful(0)
+    }
```

Comment:
these mirror the "traceDBIO*" pre-existing methods, but return `ReadAction`s instead of `ReadWriteAction`s. This helps avoid changes in method signatures that want to add tracing